Makefile

```diff
@@ -346,7 +346,16 @@ datadir/fewshot: eval/$(release)/train/user.tsv eval/$(release)/dev/user.tsv eva
 	cp eval/$(release)/dev/agent.tsv $@/agent/eval.tsv
 	touch $@
 
-datadir: datadir/agent datadir/nlg datadir/user datadir/fewshot $(all_synthetic_files)
+calibration-ood.tsv:
+	./scripts/make-calibration-ood.sh
```

Comment:
One requirement we have is to support creating datasets without S3 credentials (for testing in Travis, for third-party contributors, for people who are learning, etc.). We should check if we have s3_bucket set here, and if not, skip all the S3 business.

We should also avoid uploading the calibration-ood.tsv file to S3, given it will be part of the data directory.

Reply:
Done.